### ISSUE TITLE
pins_arduino.h issue on M5Stamp-S3

### ISSUE THREAD

**Issue by akita11** [platformio/platform-espressif32#1222]
Current pins_arduino.h for M5Stamp-S3 has some problem of SPI pin definitions, updating platformio/framework-arduinoespressif32 is required to fix it.

**Comment by Jason2866** [comment]
Are this issue already solved in Arduino espressif32 github? Platformio uses the pins Arduino.h from there.

**Comment by akita11** [comment]
Yes, already fixed on espressif32 github, not still reflected on platformio yet.
